core/src/main/java/org/apache/iceberg/StaticDataTask.java

```diff
@@ -127,7 +127,11 @@ DataFile metadataFile() {
     return metadataFile;
   }
 
-  /** @return the table rows before projection */
+  /**
+   * Returns the table rows before projection.
```

Comment:
I'm a bit confused why there is a formatting change in this PR, since spotless is still at a fixed version right? As a next step after this PR we should bump the spotless version, which most likely means that a few places in the code might change in terms of formatting

Reply:
it's because of the `Update baseline gradle plugin to 5.58.0` commit that's included here. Updating baseline wasn't possible until we dropped  Java 8 support

Reply:
moved to https://github.com/apache/iceberg/pull/10788